shared/actions/favorite.js

```diff
@@ -262,11 +267,33 @@ function _notify (state) {
   previousNotifyState = newNotifyState
 }
 
+function * _setupKBFSChangedHandler (): SagaGenerator<any, any> {
+  yield put((dispatch: Dispatch) => {
+    const debouncedKBFSStopped = _.debounce(() => {
+      const badgeAction: Action = badgeApp('kbfsUploading', false)
+      dispatch(badgeAction)
+      dispatch({type: Constants.kbfsStatusUpdated, payload: {isAsyncWriteHappening: false}})
+    }, 2000)
+
+    engine().setIncomingHandler('keybase.1.NotifyFS.FSSyncActivity', ({status}) => {
```

Comment:
i'll do the yield call stuff now. let's wait on the larger pattern until we do more integrations and see